det3d/datasets/argoverse/argoverse_common.py

```diff
@@ -0,0 +1,330 @@
+import copy
+import json
+import os
+from typing import Dict, Tuple, List
+
+import numpy as np
+import torch
+from argoverse.data_loading.object_label_record import json_label_dict_to_obj_record
+from argoverse.data_loading.pose_loader import get_city_SE3_egovehicle_at_sensor_t
+from argoverse.utils import ply_loader, se3
+from scipy import interpolate
+from scipy.spatial.transform import Rotation, RotationSpline
+
+YSize = 80
```

Comment:
Are we hard-coding these? Can we keep the same protocol as they use for nuscenes and waymo?